setup.py

```diff
@@ -48,6 +48,11 @@
             "xgboost<1.3",
             "rgf-python",
             "optuna==2.3.0",
+            "transformers==4.4.1",
+            "wandb",
+            "ray[tune]==1.2.0",
```

Comment:
use a separate option for 'nlp' to include these newly added dependencies.
install this option in linux and mac in python-package.yml